assembl/static2/js/app/components/login/sendPwdForm.jsx

```diff
@@ -17,10 +17,8 @@ class SendPwdForm extends React.Component {
   componentWillReceiveProps(nextProps) {
     const { auth } = nextProps;
     if (auth.passwordChangeRequest && auth.passwordChangeRequest.success === false) {
```

Comment:
`if (auth.passwordChangeRequest && !auth.passwordChangeRequest.success)`
Maybe you can create a dedicated method with an eloquent name and call it in the `componentWillReceiveProps`